codecov_cli/commands/labelanalysis.py

```diff
@@ -32,6 +32,7 @@
     cls=CodecovOption,
     required=True,
 )
+@click.option("--runner-name", "--runner", "runner_name", help="Runner to use")
```

Comment:
can we provide what's the default runner we use in case the user doesn't specify one?